ruby/README.md

```diff
@@ -17,7 +17,9 @@ Please refer to [Initiate an Integration with OAuth 2.0](https://developers.hubs
 
 ### Setup App
 
-Make sure you have [Docker Compose](https://docs.docker.com/compose/) installed.
+Please, make sure that you have ruby 2.6.3 installed and run bunle in your app folder.
```

Comment:
Noob question: Is ruby 2.6.3 enough for running `rails s`?

Reply:
We need rails to run rails :D
So we run bundle to install all that we need (specified in our Gemfile)